Source/controls/modifier_hints.cpp

```diff
@@ -127,14 +195,17 @@ void DrawStartModifierMenu(const Surface &out)
 
 void DrawSelectModifierMenu(const Surface &out)
 {
-	if (!select_modifier_active)
-		return;
-	if (sgOptions.Controller.bDpadHotkeys) {
-		static const CircleMenuHint DPad(/*isDpad=*/true, /*top=*/"F6", /*right=*/"F8", /*bottom=*/"F7", /*left=*/"F5");
-		DrawCircleMenuHint(out, DPad, { PANEL_LEFT + CircleMarginX, PANEL_TOP - CircleTop });
-	}
-	static const CircleMenuHint Spells(/*isDpad=*/false, "F6", "F8", "F7", "F5");
-	DrawCircleMenuHint(out, Spells, { PANEL_LEFT + PANEL_WIDTH - Spells.Width() - CircleMarginX, PANEL_TOP - CircleTop });
+    if (!select_modifier_active)
+        return;
+
+	if (sgOptions.Controller.bDpadHotkeys)
+    {
+        static const CircleMenuHint DPad(/*isDpad=*/true, /*top=*/"F6", /*right=*/"F8", /*bottom=*/"F7", /*left=*/"F5");

```

Comment:
On a seperate note, we should not hard code the keys here as they are mappable for the keyboard and X/Y/A/B for gamepads.

Reply:
i just kept it as it is :/

Reply:
Well at least your editor decided to do something to it for no reason :)

![image](https://user-images.githubusercontent.com/204594/150025122-83842f40-169b-478f-b706-7c23e5bfbe3f.png)

Reply:
yeah... i'll fix this mess

Reply:
now i guess it looks better ;)